CHANGES

```diff
@@ -3,6 +3,46 @@ repositories (changes that are automatically handled by the format upgrade tools
 are not marked). Those prefixed with "(+)" are new command/option (since
 2.1.0~alpha2).
 
+2.3.0~beta1:
+* Fix an opam 2.1 regression where the initial pin of a local VCS directory
+  would store untracked and ignored files. Those files would usually be
+  cleaned before building the package, however git submodules would not be
+  cleaned and would cause issues when paired with the new behaviour added in
+  2.3.0\~alpha1 which makes opam error when git submodules fail to update
+  (was previously a warning) [#6221 @rjbou - fix #5809]
+* Fix a regression which would make opam crash on platforms where
+  `getconf LONG_BIT` is not available (e.g. OpenBSD)
+  [#6230 @kit-ty-kate - fix #6215]
+* Fix the install cache, which was storing the wrong version of the opam file
+  after a build failure. This could be triggered easily for users with custom
+  repositories with non-populated extra-files [#6213 @kit-ty-kate]
```

Comment:
```suggestion
* Fix the installed packages internal cache, which was storing the wrong 
  version of the opam file after a build failure. This could be triggered
  easily for users with custom repositories with non-populated extra-files
  [#6213 @kit-ty-kate]
```